helpdesk/__manifest__.py

```diff
@@ -15,13 +15,15 @@
                 'Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/helpdesk',
     'depends': [
+        'sale',
```

Comment:
Is this a real dependency?
This module shouldn't depend on `sale`, it must stand by itself.

Reply:
You are rigth.
This module doesn't depend on sale